code/modules/urist/modules/shipbattles/overmapships/components/components.dm

```diff
@@ -59,56 +68,64 @@
 	strength = 750
 	health = 200
 	recharge_rate = 75
-	recharge_delay = 10 SECONDS
+	recharge_delay = 12 SECONDS
```

Comment:
I did lower the recharge rate on this lately, and with the buff to the starting laser I think also changing the recharge_delay might be a bit much. I'd go with one or the other, either the laser buff or the delay change. If you stick with the delay change, remove it for everything below freighter, because those ones are either for event ships or for alien ships/stations which are their own special beasts.